pyproject.toml

```diff
@@ -27,14 +27,13 @@ keywords = ["machine learning", "reproducibility", "visualization"]
 
 classifiers = [
   'Development Status :: 5 - Production/Stable',
-  'License :: OSI Approved :: Apache Software License',
```

Comment:
why did you remove the line about the license @reneleonhardt? Gradio is Apache 2 licensed: https://github.com/gradio-app/gradio/blob/main/LICENSE

Reply:
got it thanks